localdev/config/armada/base.yaml

```diff
@@ -0,0 +1,29 @@
+corsAllowedOrigins: ["http://localhost:3000", "http://localhost:8089"]
```

Comment:
Is this going to work as expected being localhost?

Reply:
yes it should since those will be host names used in browser